usaspending_api/common/management/commands/matview_runner.py

```diff
@@ -74,25 +90,50 @@ def generate_matview_sql(self):
             recursive_delete(self.matview_dir)
         self.matview_dir.mkdir()
 
+        if self.matview_chunked_dir.exists():
+            logger.warning("Clearing dir {}".format(self.matview_chunked_dir))
+            recursive_delete(self.matview_chunked_dir)
+        self.matview_chunked_dir.mkdir()
+
         # IF using this for operations, DO NOT LEAVE hardcoded `python3` in the command
-        exec_str = "python3 {} --quiet --dest={}/ --batch_indexes=3".format(MATVIEW_GENERATOR_FILE, self.matview_dir)
+        # Create main list of Matview SQL files
+        exec_str = f"python3 {MATVIEW_GENERATOR_FILE} --quiet --dest={self.matview_dir}/ --batch_indexes=3"
         subprocess.call(exec_str, shell=True)
 
+        # Create SQL files for Chunked Universal Transaction Matviews
+        for matview, config in self.chunked_matviews.items():
+            exec_str = f"python3 {CHUNKED_MATVIEW_GENERATOR_FILE} --quiet  --file {config['json_filepath']} --chunk-count {self.chunk_count}"
+            subprocess.call(exec_str, shell=True)
+
     def cleanup(self):
         """Cleanup files after run"""
         recursive_delete(self.matview_dir)
 
     def create_views(self):
         loop = asyncio.new_event_loop()
         tasks = []
+
+        # Create Matviews
         for matview, config in self.matviews.items():
-            logger.info("Creating Future for {}".format(matview))
+            logger.info(f"Creating Future for matview {matview}")
             sql = (self.matview_dir / config["sql_filename"]).read_text()
             tasks.append(asyncio.ensure_future(async_run_creates(sql, wrapper=Timer(matview)), loop=loop))
 
+        # Create Chunked Matviews
+        for matview, config in self.chunked_matviews.items():
+            for current_chunk in range(self.chunk_count):
+                chunked_matview = f"{matview}_{current_chunk}"
+                logger.info(f"Creating Future for chunked matview {chunked_matview}")
+                sql = (self.matview_chunked_dir / f"{chunked_matview}.sql").read_text()
+                tasks.append(asyncio.ensure_future(async_run_creates(sql, wrapper=Timer(chunked_matview)), loop=loop))
+
         loop.run_until_complete(asyncio.gather(*tasks))
         loop.close()
 
+        if "universal_transaction_matview" in self.chunked_matviews:
```

Comment:
It seems like universal_transaction_matview will always be in that list. Is there a way to not run `universal_transaction_matview`?

Originally, using the --only flag to select a specific matview to run would prevent the others from running. I don't see any logic to prevent UTM from running in this script

Reply:
Ah, I see what you meant by your original comment. I'll get this addressed